solidity/contracts/liquidity-protection/LiquidityProtection.sol

```diff
@@ -773,23 +774,20 @@ contract LiquidityProtection is ILiquidityProtection, Utils, Owned, ReentrancyGu
      *
      * @param id position id
      * @param newProvider the new provider
-     * @param target the contract to notify
-     * @param data the data to call the contract with
+     * @param callback the callback contract to notify
```

Comment:
A bit confusing imo, since 'callback' typically refers to a function.
Perhaps 'client' or something similar (TBH, 'target' wasn't all that bad).